render/wgpu/src/pixel_bender.rs

```diff
@@ -39,6 +43,57 @@ pub struct PixelBenderWgpuShader {
     staging_belt: RefCell<StagingBelt>,
 }
 
+impl PixelBenderWgpuShader {
+    /// Gets a `RenderPipeline` for the specified sample count
+    fn get_pipeline(
+        &self,
+        descriptors: &Descriptors,
+        samples: u32,
+    ) -> RefMut<'_, wgpu::RenderPipeline> {
+        let pipelines = self.pipelines.borrow_mut();
+        RefMut::map(pipelines, |pipelines| {
+            pipelines.entry(samples).or_insert_with(|| {
+                descriptors
+                    .device
+                    .create_render_pipeline(&RenderPipelineDescriptor {
+                        label: create_debug_label!("PixelBender shader pipeline").as_deref(),
+                        layout: Some(&self.pipeline_layout),
+                        vertex: VertexState {
+                            module: &self.vertex_shader,
+                            entry_point: naga_pixelbender::VERTEX_SHADER_ENTRYPOINT,
+                            buffers: &VERTEX_BUFFERS_DESCRIPTION_FILTERS,
+                        },
+                        fragment: Some(wgpu::FragmentState {
+                            module: &self.fragment_shader,
+                            entry_point: naga_pixelbender::FRAGMENT_SHADER_ENTRYPOINT,
+                            targets: &[Some(ColorTargetState {
+                                format: TextureFormat::Rgba8Unorm,
+                                // FIXME - what should this be?
+                                blend: Some(wgpu::BlendState {
```

Comment:
`None`, I imagine? Unless we're expected to blend the blend result.

Reply:
This is a pre-existing FIXME - I just moved some code around

Reply:
Got it!